components/OverflowMenu.js

```diff
@@ -64,8 +64,10 @@ class OverflowMenu extends Component {
   }
 
   getMenuPosition = () => {
-    const menuPosition = this.menuEl.getBoundingClientRect();
-    this.setState({ menuPosition });
+    if (this.menuEl) {
```

Comment:
Interesting in understanding what condition `this.menuEl` gets missing, too - One guess I can make is some kind of race condition, but what @chrisdhanaraj found may be something else.